package.json

```diff
@@ -61,7 +61,7 @@
     "vue-router": "^3.1.3",
     "vue-template-compiler": "^2.6.10",
     "vue2-leaflet": "^1.0.2",
-    "vuejs-logger": "^1.5.3",
+    "vuejs-logger": "1.5.3",
```

Comment:
Why tho?

Reply:
Latest of vuejs-logger was bugged so I fixed to that version for now
cf: https://github.com/justinkames/vuejs-logger/issues/38